frontend/src/actionCreators/mineActionCreator.js

```diff
@@ -179,6 +179,21 @@ export const fetchMineRecords = (params) => (dispatch) => {
     .finally(() => dispatch(hideLoading()));
 };
 
+export const fetchMineRecordsForMap = () => (dispatch) => {
+  dispatch(request(reducerTypes.GET_MINE_RECORDS));
```

Comment:
correct, it _should_ be different.

However, it appears he is using the same action as `fetchMineRecords` and the data is being kept in the same spot. So in this situation, the reducer being the same is fine. 


I did recommend in a comment on `Dashboard.js` to make this its own action and store the data in a different const than the `mineReducer`, the benefit being that this can be fitched when the component mounts vs when the user navigates to the tab thus delaying loading time. If that switch is made, then this would need to have its own reducerType and actionType.